config/environments/development.rb

```diff
@@ -10,12 +10,12 @@
   config.eager_load = false
 
   # Show full error reports.
-  config.consider_all_requests_local = true
+  config.consider_all_requests_local = false
```

Comment:
did a little research, is it right that this config change allows for capybara testing?